.circleci/config.yml

```diff
@@ -5,10 +5,29 @@ workflows:
   heroku_deploy:
     jobs:
       - install
-      - deploy:
+      - test:
           requires:
             - install
-jobs:
+      - deploy:
+          requires:
+            - test
+  test:
```

Comment:
Since we've already added the test job to the deploy workflow, I don't think we need a new test workflow.

Reply:
That makes sense, I'll remove the `test` workflow